workbench-core/infrastructure/src/workbenchCognito.ts

```diff
@@ -58,7 +58,10 @@ const userPoolClientDefaults: UserPoolClientOptions = {
     custom: true
   },
   preventUserExistenceErrors: true,
-  enableTokenRevocation: true
+  enableTokenRevocation: true,
+  idTokenValidity: Duration.minutes(60),
+  accessTokenValidity: Duration.minutes(60),
```

Comment:
do we have the ability to revoke tokens? if not we should consider lowering 60 mins to 5 or 15

Reply:
There is a way to revoke refresh tokens, which invalidates all access tokens generated by it. Checking if an access token has been invalidated is an expensive operation that involves a network call. Since token validation happens on every call to the api, I would lean more toward lowering the token validity length as you suggested. 15 minutes seem good?

Reply:
We may have clients that require that functionality -- lets try to keep that option open in the future but for now I am okay lowering to 15. Also when someone logs out the refresh token is invalidated right?

Reply:
yes, the refresh token is revoked and can no longer be used to generate access tokens

Reply:
lowered access and id token lifespan to 15 minutes